pkg/security/metrics/prometheus/metrics.go

```diff
@@ -0,0 +1,123 @@
+package prometheus
+
+import (
+	"sync"
+
+	"github.com/blang/semver"
+	"github.com/prometheus/client_golang/prometheus"
+	v1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	corev1listers "k8s.io/client-go/listers/core/v1"
+	"k8s.io/component-base/metrics/legacyregistry"
+	"k8s.io/klog/v2"
+
+	securityv1 "github.com/openshift/api/security/v1"
+)
+
+const (
+	separator                   = "_"
+	namespaceSCCAllocSubsystem  = "openshift_namespace_scc_allocation"
+	namespaceSCCAllocCount      = "total"
+	namespaceSCCAllocCountQuery = namespaceSCCAllocSubsystem + separator + namespaceSCCAllocCount
+)
```

Comment:
would it also make sense to collect number or percentage of uids allocated?

Reply:
Yes, definitely.